sdk/storage/README.md

```diff
@@ -167,13 +167,6 @@ async function main() {
   // SharedKeyCredential is only avaiable in Node.js runtime, not in browsers
   const sharedKeyCredential = new SharedKeyCredential(account, accountKey);
 
-  // Use RawTokenCredential with OAuth token.  You can find more
```

Comment:
We should leave a simple here for customers who want OAuth authentication.

Reply:
Maybe linking to the @azure/identity npmjs page. There's some documentation there.

Reply:
Done.